package.json

```diff
@@ -12,7 +12,7 @@
 		"bump": "lerna publish",
 		"docs:dev": "vuepress dev docs",
 		"docs:build": "vuepress build docs --debug",
-		"semantic-release": "lerna version --conventional-commits --github-release -y && lerna publish from-package -y"
+		"semantic-release": "lerna version --conventional-commits --create-release github -y && lerna publish from-package -y"
```

Comment:
the lerna version in the package.json is `^3.13.4`, that still use the '--github-release' flag: 
https://github.com/lerna/lerna/tree/v3.13.4/commands/version

Reply:
Good spot. Updated to `lerna@3.22.1` as well.